src/main/java/graphgui/GraphPane.java

```diff
@@ -225,4 +229,344 @@ public void edgeChanged(Edge edge) {
+        if (tokens.length < 2) {
+          this.controller.appendTextArea("Nespravny pocet parametrov pre delete\n");
+          break;
+        }
+
+        switch (tokens[1]) {
+          case "edge": {
+            if (tokens.length < 4) {
+              this.controller.appendTextArea("Nespravny pocet parametrov pre remove edge\n");
+              break;
+            }
+              
+            int from;
+            int to;
+            try {
+              from = Integer.parseInt(tokens[2]);
+              to = Integer.parseInt(tokens[3]);
+            } catch (NumberFormatException e) {
+              this.controller.appendTextArea(
+                  "Parametre pre remove edge maju nespravny format - ocakavaju sa dve cele cisla\n"
+              );
+              break;
+            }
+
+            try {
+              this.graph.removeEdge(from, to);
+            } catch (IllegalArgumentException e) {
+              this.controller.appendTextArea(String.format("%s\n", e.toString()));
+              break;
+            } catch (IndexOutOfBoundsException e) {
+              this.controller.appendTextArea("Neplatny index vrcholu From alebo To\n");
+              break;
+            }
+              
+            this.controller.appendTextArea(
+                String.format("Hrana from %d to %d uspesne odstranena\n", from, to)
+            );                      
+            break;
+          }
+          case "vertex": {
+
+            if (tokens.length != 3) {
+              this.controller.appendTextArea(
+                  String.format("delete vertex ocakava 1 parameter, nasiel %d\n", tokens.length - 2)
+              );
+              break;
+            }
+
+            int vertexId;
+
+            try {
+              vertexId = Integer.parseInt(tokens[2]);
+            } catch (NumberFormatException e) {
+              this.controller.appendTextArea(
+                  String.format("delete vertex ocakava int vertexId, nasiel %s\n", tokens[2])
+              );
+              break;
+            }
+
+            try {
+              this.graph.removeVertex(this.graph.getVertex(vertexId));
+              this.controller.appendTextArea(
+                  String.format("Vrchol %d uspesne zmazany\n", vertexId)
+              );
+            } catch (IndexOutOfBoundsException e) {
+              this.controller.appendTextArea(String.format("vertexId %d je neplatne\n", vertexId));
+            }
+            break;
+          }
+          default: {
+            this.controller.appendTextArea(
+                String.format(
+                    "Druhy parameter delete moze byt 'vertex' alebo 'edge', nie '%s'\n",
+                    tokens[1]
+                )
+            );
+            break;
+          }
+        }
+
+        break;
+      }
+      case "move": {
+        if (tokens.length != 4) {
+          this.controller.appendTextArea(
+              "move vyzaduje parametre int vertexId double newX double newY\n"
```

Comment:
Nebolo by lepšie to dať `move vyzaduje parametre (int)vertexId (double)newX (double)newY` ?